layouts/rss.xml

```diff
@@ -28,19 +31,18 @@
     {{ with .OutputFormats.Get "RSS" }}
     {{ printf "<atom:link href=%q rel=\"self\" type=%q />" .Permalink .MediaType | safeHTML }}
     {{ end }}
-    {{ range $pages }}
+    {{ range $pages }}{{ if ne .Params.rss_ignore true }}
     <item>
       <title>{{ .Title }}</title>
       <link>{{ .Permalink }}</link>
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
-      {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
+      {{ with .Site.Author.email }}<author>{{.}}{{ with .Site.Author.name }} ({{.}}){{end}}</author>{{end}}
```

Comment:
Mmmm... we originally have `$.Site.Author.name` because the default rss  template also have this. But it certainly seems to make sense to change this. Perhaps you could consider sending a PR to the main hugo repo and see what the authors say about this?

https://github.com/gohugoio/hugo/blob/65e52a7f5c79e6cfeabafa67d1bff4994c7fc4a4/tpl/tplimpl/embedded/templates/_default/rss.xml#L33

Reply:
ok, I will do that. Before that, I temporarily revert the change.

Reply:
https://github.com/gohugoio/hugo/pull/10065#issuecomment-1173594654

The Hugo maintainer said that using `$` makes no functional difference but will make it much clearer.